models/schema_site_anp_epg_private.go

```diff
@@ -0,0 +1,40 @@
+package models
```

Comment:
Why a new file and not add the private link label to the existing site_anp_epg schema?

Reply:
The existing site_anp_epg schema is used by other resources too which don't have private link label